docs/development/workflow/development_workflow.rst

```diff
@@ -250,7 +250,7 @@ details.
 Fetch the latest Astropy
 ************************
 
-From time to time you should fetch the development version (i.e. Astropy
+From time to time you should fetch the development version (i.e., Astropy
 ``astropy/main``) changes from GitHub::
 
    git fetch astropy --tags
```

Comment:
I don't think `--tags` is particularly useful: it just adds noise to the branch list.

Reply:
This is actually very important because if you don't have the latest tags from `astropy` the *version* that gets assigned to your development copy of `astropy` will be wrong. I.e. if you were missing the `v5.3` tag, then in your local version of `astropy` might indicate that its `v5.2.3.dev....`. This can be an issue in unexpected cases.

Reply:
Yes, we should keep `--tags` for the reason that William stated above.